internal/godplugin/job/job.go

```diff
@@ -30,20 +31,20 @@ Done:
 	for {
 
 		sleep := j.nextIn()
-		j.Debugf("sleeping for %s to reach frequency of %d sec", sleep, j.UpdEvery)
+		j.Debugf("sleeping for %s to reach frequency of %d sec", sleep, j.C.UpdEvery)
 		time.Sleep(sleep)
 
-		j.curRun = time.Now()
-		if !j.lastRun.IsZero() {
-			j.sinceLast.Duration = j.curRun.Sub(j.lastRun)
+		j.t.curRun = time.Now()
```

Comment:
https://golang.org/pkg/time/#Ticker 
I think use Ticker rather than timers is simpler and Go style.
We can refactor it in another PR :)

Reply:
Yeah, different story. Actually we have to use monotonic to calc next run and real time to measure  duration `sinceLast`